eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/EventHubClientImpl.java

```diff
@@ -25,7 +39,11 @@
     /**
      * It will be truncated to 128 characters
      */
-    public static String USER_AGENT = null;
+    private static String userAgent = null;
+
+    public static String getUserAgent() {
+        return userAgent;
+    }
```

Comment:
as noted previously - don't do this change, please revert and suppress as necessary.